content/documentation/concepts/nats-queueing.md

```diff
@@ -11,10 +11,11 @@ category = "concepts"
 
 # NATS Queueing
 
-NATS supports message queueing using point-to-point (one-to-one) communication.
+NATS supports message queueing using point-to-point (one-to-one) communication. Although NATS at it's core is 'fire and forget', message queueing can be used as a performance optimizer.
+
 
 To create a message queue, subscribers register a queue name. All subscribers with the same queue name form the queue group. As messages on the registered subject are published, one member of the group is chosen randomly to receive the message. Although queue groups have multiple subscribers, each message is consumed by only one.
```

Comment:
"To create a queue subscription"

Reply:
I would also add at the end "Queue subscribers are ideal for auto scaling as you can add or remove them anytime, without any configuration changes or restarting the server or clients."

Reply:
resolved